app/src/test/java/org/simple/clinic/newentry/PatientEntryEffectHandlerTest.kt

```diff
@@ -100,6 +100,20 @@ class PatientEntryEffectHandlerTest {
     verifyZeroInteractions(validationActions)
   }
 
+  @Test
+  fun `when fetch colony or village names effect is received, then fetch them from the database`() {
+    // given
+    val colonyOrVillages = listOf("colony1", "colony2", "colony3", "colony4")
+    whenever(patientRepository.allColoniesOrVillagesInPatientAddress()).thenReturn(colonyOrVillages)
+    // when
+    setupTestCase()
+    testCase.dispatch(FetchColonyOrVillagesEffect)
+
+    //then
+    testCase.assertOutgoingEvents(ColonyOrVillagesFetched(colonyOrVillages))
+    verifyNoMoreInteractions(ui)
```

Comment:
This should be `verifyZeroInteractions`. When we are asserting to make sure no UI interactions are happening we use `verifyZeroInteractions`. We use `verifyNoMoreInteractions` if want to make that after certain interactions there are no more interactions for the given class.